01_science_technology_and_epistemology/01_science_technology_and_epistemology.jl

```diff
@@ -8,72 +8,186 @@ using InteractiveUtils
+So then, what role does Science play in all this and why is it useful for technology? As we already said, it is a tool that technology has to formalize all that chaotic based knowledge discovery. 
+Formalization is very useful to teach and communicate some base knowledge and to build a solid foundation in order to keep expanding it. 
+But it is also the way we found to draw out subjectivities regarding the understanding of our world.
 
-Take, for example, the physics field. A field that undoubtedly contributed enormously to the development of humanity. But now, it seems to have started a new romance. Some modern theoretical physicists seem to have ended their relationship with experimentation because of a crazy love for 'mathematical beauty'. Tons and tons of papers, and zero experiments. It is somewhat worrying. And anyway, why should the laws of our complex reality be "elegant"?
-
-An interesting concept related to this is that of the low hanging fruits. As the name suggests, it refers to scientific laws that are the ones with the simplest explanations, simplest equations and elegant conclusions. Although simple, this doesn't mean they don't require a stroke of genius to come up with, don't get us wrong. But naturally, this were the first discoveries made, making a lot of assumptions, and imposing various constraints. In this way, it is quite natural to think of beautiful equations being derived that express laws of nature. But this concept of mathematical beauty can sometimes be taken too far, seeking and trying to impose it to all scientific discovery when in reality it is a human construct.
+## What is Science?
 
-And there are fields of science apart from physics, like biology or chemistry, that are examples of fields that don't have any of these "beautiful" and extremely precise equations. In the rest of science, we have what are called "emergent laws" that arise from very complicated systems wherewhere the individual details don't matter, but the system as a whole behave in a very specific and recognizable way.
+As a first approach, science is a method that was conceived to help humans discover the principles -invariant laws- that describe the world. 
+And note that we said "method", because this is all it really is: A methodology which ensures that we are as objective and data-driven as possible.
+
+And it is common to think that one does this by observation. 
+We observe, observe, and observe, and we look for theories to best explain the mass of facts. But this idea began to annoy [Sir Karl Popper](https://en.wikipedia.org/wiki/Karl_Popper) (1902-1994), the scientific philosopher, more and more. 
+The problem, as Popper saw it, is that some bodies of knowledge more properly named pseudosciences would be considered scientific if the “Observe & Deduce” operating definition were left alone. 
```

Comment:
> were left alone. 

what does this mean?

Reply:
if we only use the “Observe & Deduce” definition (leaving the ability of a theory to be falsified) lots of pseudoscience would we consider Scientific.